packages/content-source-maps/src/types.ts

```diff
@@ -49,13 +61,14 @@ export type WidgetNamespace =
   | 'app'
   | 'editor-builtin';
 
-interface EditorInterfaceSource {
+export interface EditorInterfaceSource {
   widgetId: WidgetId;
   widgetNamespace: WidgetNamespace;
 }
 
 interface ContentSourceMaps {
-  version: number;
+  sys: { type: 'ContentSourceMaps' };
+  version: number; //@TODO remove once we remove version from GraphQL API: https://contentful.atlassian.net/browse/TOL-2016
```

Comment:
we could remove it directly, if more data is coming from the API it should not break 🤔